src/db/migration/migrate20220729-FilesShareWithMeFolder.pl

```diff
@@ -20,42 +20,47 @@
 #
 # Add Files Shared With me system folder (id = 20) to each mailbox.
 #
-
 use strict;
 use Migrate;
 
-sub bumpUpMailboxChangeCheckpoints();
-
-my $CONCURRENCY = 10;
-my $ID = 20;
-my $METADATA = 'd1:ai1e3:das5:false4:mseqi1e4:unxti21e1:vi10e2:vti8ee';
-my $NOW = time();
-my $FOLDERNAME = 'Files shared with me';
-
-bumpUpMailboxChangeCheckpoints();
+# Verify Schema Version Number
+Migrate::verifySchemaVersion(116);
 
-my @sqlRename;
-my %mailboxes = Migrate::getMailboxes();
+checkAndCreateFilesSharedWithMeFolder();
 
-my %uniqueGroups;
-foreach my $gid (values %mailboxes) {
-    if (!exists($uniqueGroups{$gid})) {
-        $uniqueGroups{$gid} = $gid;
-    }
-}
-my @sqlInsert;
-my @groups = sort(keys %uniqueGroups);
-foreach my $gid (sort @groups) {
-    my $sql = createFolder($gid);
-    push(@sqlInsert, $sql);
-}
-Migrate::runSqlParallel($CONCURRENCY, @sqlInsert);
+# Update Schema Version Number
+Migrate::updateSchemaVersion(116, 117);
 
 exit(0);
 
-
 #####################
 
+sub checkAndCreateFilesSharedWithMeFolder() {
+    my $CONCURRENCY = 10;
+    my $FOLDERID = 20;
+    my $METADATA = 'd1:ai1e3:das5:false4:mseqi1e4:unxti21e1:vi10e2:vti8ee';
+    my $NOW = time();
+    my $FOLDERNAME = 'Files shared with me';
+
+    bumpUpMailboxChangeCheckpoints();
+
+    my %mailboxes = Migrate::getMailboxes();
+    my %uniqueGroups;
+    foreach my $gid (values %mailboxes) {
+        if (!exists($uniqueGroups{$gid})) {
+            $uniqueGroups{$gid} = $gid;
+        }
```

Comment:
```suggestion
        $uniqueGroups{$gid} = $gid;
```
No need to check exists here.

Reply:
I just did copy/paste from existing script, so don't have much idea if this is needed or not
can we defer this to next patch release as an improvement?

Reply:
> I just did copy/paste from existing script, so don't have much idea if this is needed or not can we defer this to next patch release as an improvement?

No need to "fix" it, really. Just pointing it out.

Reply:
thanks